@nrwl/workspace migrations 13-10-0

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
<!-- What is the behavior that currently you experience? -->
The `@nrwl/workspace migrations 13-10-0` are not applied when following the unpdate procedure, despite the fact that the terminal reports success.
One migration seems incorrect, see the explanation below.

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->
The `@nrwl/workspace migrations 13-10-0` are applied when following the update procedure.

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->
Checkout this commit https://github.com/rfprod/nx-ng-starter/tree/e7d4acb71e29b02cbaa628747c7c68458856e484 and follow the procedure

0. instal the project dependencies first
```bash
yarn install
```
1. migrate
```bash
npx nx migrate @nrwl/workspace
```
2. install
```bash
yarn install
```
3. apply migrations
```bash
npx nx migrate --run-migrations
```
4. verify the result
4.1 After applying the migrations the `decorate-angular-cli.js` file is the same as previously.
4.2 After applying the migrations the `nx.json` file is the same as previously (except formatting which does not matter in the given context).

## Additional context

As I see in the `migrations.json`, two things should be migrated
```json
{
  "migrations": [
    {
      "version": "13.10.0-beta.0",
      "description": "Update the decorate-angular-cli script to require nx instead of @nrwl/cli",
      "cli": "nx",
      "implementation": "./src/migrations/update-13-10-0/update-decorate-cli",
      "package": "@nrwl/workspace",
      "name": "13-10-0-update-decorate-cli"
    },
    {
      "version": "13.10.0-beta.0",
      "description": "Update the tasks runner property to import it from the nx package instead of @nrwl/worksapce",
      "cli": "nx",
      "implementation": "./src/migrations/update-13-10-0/update-tasks-runner",
      "package": "@nrwl/workspace",
      "name": "13-10-0-update-tasks-runner"
    }
  ]
}
```

When I checked the source code of the migrations and applied it manually **almost** everything worked.
There is possibly a bug in one migration (see below).

**The migration that worked manually (according to the instructions)**
I migrated the `decorate-angular-cli.js` manually, and it seems to work fine.
I replaced the substring manually according to this migration https://github.com/nrwl/nx/blob/master/packages/workspace/src/migrations/update-13-10-0/update-decorate-cli.ts.
I replaced the substring `@nrwl/cli/lib/decorate-cli` with the substring `nx/src/adapter/decorate-cli` in this file https://github.com/rfprod/nx-ng-starter/blob/e7d4acb71e29b02cbaa628747c7c68458856e484/decorate-angular-cli.js#L64 like this
```javascript
try {
  symlinkNgCLItoNxCLI();
  require('nx/src/adapter/decorate-cli').decorateCli();
  output.log({ title: 'Angular CLI has been decorated to enable computation caching.' });
} catch (e) {
  output.error({ title: 'Decoration of the Angular CLI did not complete successfully' });
}
```

**The migration that did not work manually (according to the instructions)**
I migrated the `nx.json` manually, and it did not work **at first**.
Then I took a closer look at what should be modified, and it seems like there is a typo in the migration https://github.com/nrwl/nx/blob/master/packages/workspace/src/migrations/update-13-10-0/update-tasks-runner.ts
At first, I replaced the substring `@nrwl/workspace/tasks-runner/default` with the usbstring `nx/tasks-runner/default` in this file https://github.com/rfprod/nx-ng-starter/blob/e7d4acb71e29b02cbaa628747c7c68458856e484/nx.json#L87.
After this change the `nx` commands started failing with the following error

```plaintext
Error: Cannot find module 'nx/tasks-runner/default'
Require stack:
- /media/suser/DATA/git/rfprod/nx-ng-starter/node_modules/nx/src/tasks-runner/run-command.js
- /media/suser/DATA/git/rfprod/nx-ng-starter/node_modules/nx/src/command-line/run-one.js
- /media/suser/DATA/git/rfprod/nx-ng-starter/node_modules/nx/src/command-line/nx-commands.js
- /media/suser/DATA/git/rfprod/nx-ng-starter/node_modules/nx/bin/init-local.js
- /media/suser/DATA/git/rfprod/nx-ng-starter/node_modules/nx/bin/nx.js
...
```

because there is no such module indeed.
But there is `nx/tasks-runners/default` - notice the `s`, task-runner**s** not task-runner.
This configuartion worked

```json
"tasksRunnerOptions": {
    "default": {
      "options": { ... },
      "runner": "nx/tasks-runners/default"
    }
  },
```

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->

### Failure Logs
<!-- Please include any relevant log snippets or files here. -->
The were no failures during the automatic migration, but nothing has changed after applying the migrations which is possibly the unwanted behavior.
The terminal output
```plaintext
>  NX  Running migrations from 'migrations.json'

Running migration 13-10-0-update-decorate-cli
Successfully finished 13-10-0-update-decorate-cli
---------------------------------------------------------
Running migration 13-10-0-update-tasks-runner
Successfully finished 13-10-0-update-tasks-runner
---------------------------------------------------------

>  NX  Successfully finished running migrations from 'migrations.json'
```

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->
```bash
$ nx report

 >  NX   Report complete - copy this into the issue template

   Node : 16.14.1
   OS   : linux x64
   yarn : 1.22.18
   
   nx : 13.10.1
   @nrwl/angular : 13.10.1
   @nrwl/cypress : 13.10.1
   @nrwl/detox : Not Found
   @nrwl/devkit : 13.10.1
   @nrwl/eslint-plugin-nx : 13.10.1
   @nrwl/express : Not Found
   @nrwl/jest : 13.10.1
   @nrwl/js : 13.10.1
   @nrwl/linter : 13.10.1
   @nrwl/nest : 13.10.1
   @nrwl/next : Not Found
   @nrwl/node : 13.10.1
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 13.10.1
   @nrwl/web : 13.10.1
   @nrwl/workspace : 13.10.1
   typescript : 4.6.3
   rxjs : 7.5.5
   ---------------------------------------
   Community plugins:
         apollo-angular: 3.0.1
         @ngxs/schematics: 0.0.1-alpha.5
         @storybook/angular: 6.5.0-alpha.34
```

I hope the bug report helps.

Thank you!

--- a/packages/workspace/src/migrations/update-13-10-0/update-tasks-runner.ts
+++ b/packages/workspace/src/migrations/update-13-10-0/update-tasks-runner.ts
@@ -9,9 +9,9 @@ export function updateTasksRunner(host: Tree) {
   if (
     config?.tasksRunnerOptions['default'] &&
     config?.tasksRunnerOptions['default'].runner ==
-      '@nrwl/workspace/tasks-runner/default'
+      '@nrwl/workspace/tasks-runners/default'
   ) {
-    config.tasksRunnerOptions['default'].runner = 'nx/tasks-runner/default';
+    config.tasksRunnerOptions['default'].runner = 'nx/tasks-runners/default';
   }
   updateWorkspaceConfiguration(host, config);
 }

